src/ScriptCs.ComponentModel.Composition/ScriptCsCatalog.cs

```diff
@@ -171,6 +173,14 @@ public ScriptCsCatalog(string path, string searchPattern, IFileSystem fileSystem
                 throw new ArgumentNullException("fileSystem");
             }
 
+            if (scriptArgs == null)
+            {
+                scriptArgs = new string[0];
+            }
+
+            System.Diagnostics.Debug.WriteLine("Foo");
```

Comment:
LOL. Yes, I'll delete it. I put then when I was trying to debug, because i had some weirdness. Anyway, commit coming, good catch :p